docs/components/compact.md

```diff
@@ -49,6 +49,8 @@ Ideally, you will have equal retention set (or no retention at all) to all resol
 
 Not setting this flag, or setting it to `0d`, i.e. `--retention.resolution-X=0d`, will mean that samples at the `X` resolution level will be kept forever.
 
+Optionally, you can also retain blocks which are compacted into larger blocks. You can supply `--objstore-retain-compacted-blocks-to-bucket.config` and Thanos will move compacted blocks to the specified bucket. This can be useful if you do not want to lose blocks during rewrites, which can happen in compaction, delete series etc.
```

Comment:
it's not clear-- does this apply to only the original blocks, or also to higher-order compactions?

if the goal is to keep a backup of the original prometheus blocks, it seems the latter is not necessary